feature-toggle-api/templates/cronjob.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1beta1
```

Comment:
should this always be enabled ? and is cronjob the best name or should it be something like importer.yaml?

Reply:
agreed - will make optional and change name